src/app/sentry/init.ts

```diff
@@ -0,0 +1,14 @@
+import * as Sentry from "@sentry/browser"
+
+export default () => {
+  const dsn = process.env.REACT_APP_SENTRY_DSN
+  const environment = process.env.REACT_APP_SENTRY_ENVIRONMENT
```

Comment:
This was renamed to `REACT_APP_ENVIRONMENT`

Reply:
Yes, I see it now, thanks